Dockerfile

```diff
@@ -1,4 +1,30 @@
+#-----------------------------------------------------#
+#                     System Base                     #
+#-----------------------------------------------------#
 FROM openjdk:8u171-alpine3.7
 RUN apk --no-cache add curl
-COPY target/oncostore*.jar oncostore.jar
-CMD java ${JAVA_OPTS} -jar oncostore.jar
\ No newline at end of file
+
+#-----------------------------------------------------#
+#                Install Variantstore                 #
+#-----------------------------------------------------#
+# Define install directory
+ENV variantstore=/root/variantstore
+RUN mkdir ${variantstore}
+
+# Get required files for setup
+COPY ./src ${variantstore}/src
+COPY ./pom.xml ${variantstore}/pom.xml
+COPY ./micronaut-cli.yml ${variantstore}/micronaut-cli.yml
+COPY ./.mvn ${variantstore}/.mvn
+COPY ./mvnw ${variantstore}/mvnw
+COPY ./mvnw.cmd ${variantstore}/mvnw.cmd
```

Comment:
Could you comment on the reason for creating a new directory and copying the content? I guess there's a reason behind it. :)

Reply:
Just for better overview in the container.
Often, especially if you install or compile software in a container, you end up with various temporary/configuration/meta data in the home directory of the container. A separate variantstore directory allows clear distinction which files belong to the variantstore and which one does not.